### DISCUSSION TITLE
2406 data release

### DISCUSSION THREAD

**Discussion by julesjacobsen** [exomiser/Exomiser#562] [Data release]
2406 data update:

These data files are *only* compatible with the Exomiser [14.0.0 release](https://github.com/exomiser/Exomiser/releases/tag/14.0.0). Attempting to run them with an earlier version of the software will result in an error and Exomiser not starting.

Highlighted changes for this release includes the updated gnomAD v4.1 data for hg38, updated ClinVar annotations to fix incorrect `NOT_PROVIDED` status (issue #560) and inclusion of missing SpliceAI scores (issue #546). Note that only SpliceAI scores for SNPs >= 0.1 have been included. For the phenotype data, the pairwise phenotype similarities and IC calculations have been produced using the Monarch Initiative's Phenomics Integrated Ontology - [Phenio](https://github.com/monarch-initiative/phenio) and the new [semsimian](https://github.com/monarch-initiative/semsimian/) library.

Following our testing, we recommend updating your Exomiser analysis YAML files to include `ALPHA_MISSENSE` in the `pathogenicitySources` field:

```yaml
pathogenicitySources: [ REVEL, MVP, ALPHA_MISSENSE ]
```

Direct links:

[2406_hg19](https://g-879a9f.f5dc97.75bc.dn.glob.us/data/2406_hg19.zip)
[2406_hg38](https://g-879a9f.f5dc97.75bc.dn.glob.us/data/2406_hg38.zip)
[2406_phenotype](https://g-879a9f.f5dc97.75bc.dn.glob.us/data/2406_phenotype.zip)

Exomiser 2406 database data sources
==

Genomic Data
===

Transcripts:
 - ensembl 87 (GRCh37)
 - ensembl 111 (GRCh38, MANE)

Frequency sources:
 - gnomAD exomes r2.1.1 (GRCh37), r4.1 (GRCh38)
 - gnomAD genomes r2.1.1 (GRCh37), r4.1 (GRCh38)
 - ESP6500SI-V2-SSA137.GRCh38-liftover
 - UK10K_COHORT.20160215
 - dbSNP 156

 - gnomAD-SV v2.1
 - dbVar 2023-09-18
 - DGV 2020-02-25
 - GONL 2016-10-13
 - DECIPHER 2015

Pathogenicity sources:
 - dbNSFP 4.5a (Polyphen, MutationTaster, SIFT, MVP, REVEL, AlphaMissense, SpliceAI) _n.b. SpliceAI annotations are free for academic and not-for-profit use; other use requires a commercial license from Illumina, Inc. _
 - ClinVar 2024-06-11

Phenotype data 
===
 - OMIM 2024-06-12
 - Orphanet 2023-12-04
 - HPO 2024-04-26
 - HPO annotations 2024-04-26
 - IMPC release 21.0 
 - MP 2024-05-28
 - Zfin downloaded 2024-06-13
 - ZP downloaded 2024-06-13

**Comment by lacek** [comment]
Will the new 2406 data files be available in https://data.monarchinitiative.org/exomiser/data/?

**Comment by ewamat633** [comment]
Can you please check that the direct links to 2406 datasets above are working correctly as I am having problems downloading the 2406 data?
`wget: unable to resolve host address ‘g-879a9f.f5dc97.75bc.dn.glob.us`
